.github/workflows/autocommit.yml

```
@@ -0,0 +1,21 @@
+name: Commit to keep branch open
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  check-commits:
+    runs-on: ubuntu-20.04
```

Change this to `ubuntu-latest` or `ubuntu-22.04`. No point in running on an old image, and the job is simple enough that it's unlikely to break.

```suggestion
    runs-on: ubuntu-latest
```